jenkins/bootstrap_test.py

```diff
@@ -903,15 +903,6 @@ def testCIPaths(self):
             self.assertFalse(any(
                 PULL in o for o in (path.a, path.kw)))
 
-    def testNoFinishWhenStartFails(self):
```

Comment:
nit: consider adding a one line description since whatever lameo wrote this test forgot to do so.